dashboard/src/app/sitemap.ts

```diff
@@ -0,0 +1,102 @@
+import { MetadataRoute } from 'next';
+import { env } from '@/lib/env';
+
+export default function sitemap(): MetadataRoute.Sitemap {
+  const baseUrl = env.NEXT_PUBLIC_BASE_URL;
+
+  return [
+    /****************** PUBLIC PAGES ******************/
+
+    {
+      url: baseUrl,
+      lastModified: new Date(),
+      changeFrequency: 'monthly',
```

Comment:
Should some of these be set to a higher change frequency? I imagine that our docs/landing page will have weekly updates for the next while. Thoughts?